standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -932,6 +935,10 @@ public static Configuration newMetastoreConf() {
     if (getBoolVar(conf, ConfVars.SCHEMA_VERIFICATION)) {
       setBoolVar(conf, ConfVars.AUTO_CREATE_ALL, false);
     }
+
+    if (getBoolVar(conf, ConfVars.DUMP_CONFIG_ON_CREATION) && LOG.isInfoEnabled()) {
```

Comment:
Wouldn't it be better to print this once per JVM?  I don't think the config file is ever re-read and this would avoid flooding the log.